pkg/core/resources/store/error.go

```diff
@@ -0,0 +1,57 @@
+package store
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/kumahq/kuma/pkg/core/resources/model"
+)
+
+var (
+	ErrorInvalidOffset = errors.New("invalid offset")
```

Comment:
```suggestion
	ErrInvalidOffset = errors.New("invalid offset")
```

please make naming consistent with other errors